otterdog/eclipse-volttron.jsonnet

```diff
@@ -49,6 +49,12 @@ orgs.newOrg('eclipse-volttron') {
       secret_scanning_push_protection: "disabled",
       web_commit_signoff_required: false,
     },
+    orgs.newRepo('eclipse-volttron.github.io') {
+      allow_update_branch: false,
+      secret_scanning: "disabled",
```

Comment:
In general it is advised that for new repos secret scanning and push protection should be enabled. If you remove these lines, it would be enabled.

For all existing repos these settings should also be enabled, but this can be done in a separate PR at a later time.